src/rebar_prv_dialyzer.erl

```diff
@@ -23,6 +23,7 @@
 -spec init(rebar_state:t()) -> {ok, rebar_state:t()}.
 init(State) ->
     Opts = [{update_plt, $u, "update-plt", boolean, "Enable updating the PLT. Default: true"},
+            {update_base_plt, $b, "update-base-plt", boolean, "Enable updating the base PLT. Default: true"},
```

Comment:
I don't think this a suitable option name as it does not quite match the other arguments. This option implies the following workflow:
* update base plt (if enabled)
* update project plt (if enabled)
* update success typing (if enabled)

But really we only update the base plt if the project plt does not exist. I think it would better to use a name that describes whether we can write to the base PLT, or only read from it.